repos/system_upgrade/el7toel8/actors/scanmemory/tests/test_scanmemory.py

```diff
@@ -0,0 +1,39 @@
+import os
+
+from leapp.libraries.actor import library
+from leapp.libraries.common import testutils
+from leapp.libraries.stdlib import api
+from leapp.models import MemoryInfo
+
+
+class mocked_get_memoryinfo(object):
+    def __init__(self, filename):
+        self.filename = os.path.join('tests', 'files', filename)
+        self._orig_get_memoryinfo = library._get_memoryinfo
+
+    def __call__(self):
+        """
+        Return lines of the self.filename test file located in the files directory.
+
+        Those files contain /proc/meminfo content from various machines.
+        """
+        return self._orig_get_memoryinfo(self.filename)
+
+
+def test_low_memory(monkeypatch):
+    mocked_memoryinfo = mocked_get_memoryinfo('meminfo_low')
```

Comment:
Actually, that is on many places in other actors as well, so I am ok with that. But yes, it could be eg. parametrized using decorator, which would do the whole thing without need of two almost same functions.